### ISSUE TITLE
Dark mode not working in v4

### ISSUE THREAD

**Issue by tarifalhasan** [nativewind/nativewind#916]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Expo Snack**
If applicable, add a link to an Expo snack demonstrating the issue. There are multiple example snacks available at nativewind.dev.

**Screenshots**

**Additional context**

**Comment by tyrauber** [comment]
Here is a dark mode example with custom colors:

```
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer base {
  :root {
    --background: 40 30% 96%;
    --foreground: 40 5% 12%;

  }
  .dark:root {
    --background: 40 5% 12%;
    --foreground: 40 30% 96%;
  }

  .default-bg {
    @apply background dark:background;
  }
}
```
```
module.exports = {
  darkMode: 'class',
  content: ['./src/**/*.{ts,tsx,svg}'],
  presets: [require('nativewind/preset')],
  theme: {
    extend: {
      colors: {
        background: 'hsl(var(--background))',
        foreground: 'hsl(var(--foreground))',
      }
    },
  },
};
```

**Comment by danstepanov** [comment]
Closed as duplicate of #917
